charts/app-config-frontend/values.yaml

```diff
@@ -2,23 +2,34 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
+# -- Number of Appconfig frontend replicas to deploy
+replicaCount: 2
```

Comment:
Why the default is set to 2? Wouldn't it add more load to instances running on a single-host?

Reply:
The default configuration is for clustered setup